pr_agent/algo/__init__.py

```diff
@@ -66,10 +66,10 @@
     'claude-3-5-sonnet': 100000,
     'groq/llama3-8b-8192': 8192,
     'groq/llama3-70b-8192': 8192,
+    'groq/llama-3.1-8b-instant': 8192,
+    'groq/llama-3.3-70b-versatile': 8192,
```

Comment:
why 8192 ?

from your link:

<img width="861" alt="image" src="https://github.com/user-attachments/assets/66400b5a-2e69-48e1-8590-281fb49dc139" />

Reply:
make sure the tokens are correct. 8192 seems small to a modern model

Reply:
I retrieved the max tokens from the configuration of LiteLLM:
https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json

Reply:
"max_tokens" is max_input_tokens

![image](https://github.com/user-attachments/assets/3648a3e6-e5f6-44d7-a14c-1483116be378)

Reply:
Okay, I updated it!

Reply:
please review and fix the tokens for the other models

Reply:
I've already reviewed them.

Reply:
'groq/llama-3.1-8b-instant': 8192,

Reply:
![image](https://github.com/user-attachments/assets/20cdb726-371d-465e-a196-1fa6ee75583a)

Reply:
from the main link you gave

Reply:
https://github.com/BerriAI/litellm/blob/24dd6559a68ddf2bbb9642c48c3482d0058b3e75/model_prices_and_context_window.json#L2159-L2161